sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/credential/TokenRequestContextExperimental.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.experimental.credential;
+
+import com.azure.core.credential.TokenRequestContext;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Contains details of a request to get a token.
+ */
+public class TokenRequestContextExperimental extends TokenRequestContext {
```

Comment:
It would be good to give this a name that will eventually be used when it's moved to `azure-core`.

Reply:
This will merge with TokenRequestContext when GA happens